protocol/query/method/message/data/dataFederationResult.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataFederationResult.json",
+    "description": "Sent by an server to a remote server, to inform them about the result of the authentication procedure",
+    "type": "object",
+    "oneOf": [
+        {
+            "type": "object",
+            "properties": {
+                "object": {
+                    "const": "federation"
+                },
+                "action": {
+                    "const": "result"
+                },
+                "status": {
+                    "type": "string",
+                    "$comment": "status of the authentication attempt"
+                },
```

Comment:
The status value could be restricted to be some specific value ("failed", etc ?), either with pattern or with enum.
Or with const if only one value possible.

Reply:
should be resolved in https://github.com/dedis/popstellar/pull/1832/commits/0929ebdb329ed2c75639139d90b9bfb5da64f0b0